homeassistant/components/ecowitt/config_flow.py

```diff
@@ -0,0 +1,80 @@
+"""Config flow for ecowitt."""
+from __future__ import annotations
+
+import logging
+import secrets
+from typing import Any
+
+from aioecowitt import EcoWittListener
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_PATH, DEFAULT_PORT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_PATH, default=f"/{secrets.token_urlsafe(16)}"): cv.string,
```

Comment:
What is the path used for? Any reason this should be chosen by the user? Can we make it the config entry ID.

Also, the default here will be the same for each user